projects/dex-ui/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "dex-ui",
+  "description": "A user interface for Beanstalk Dex",
+  "private": true,
+  "version": "1.0.0",
+  "license": "MIT",
+  "scripts": {
+    "start": "vite --port 2424",
+    "build": "vite build",
+    "lint": "npx eslint src",
+    "lint:fix": "npm run lint -- --fix",
+    "prettier": "npx prettier src --check",
+    "prettier:fix": "npm run prettier -- --write",
+    "format": "npm run prettier:fix && npm run lint:fix"
+  },
+  "dependencies": {
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
+  },
+  "devDependencies": {
```

Comment:
add lint-staged and a .lintstagedrc (or whatever). see the other projects for reference

Reply:
I assume you mean with husky, will add.

Reply:
I realized husky and the lint-staged pre-commit hook isn't working even in the UI project. We can split into a separate PR against master if we wanted to.